packages/smart-contracts/test/contracts/BatchNoConversionEthPayments.test.ts

```diff
@@ -208,52 +218,6 @@ describe('contract: batchNoConversionPayments: Ethereum', () => {
 
       expect(await provider.getBalance(batchAddress)).to.be.equal(0);
     });
-
-    it('Should revert batch if input s arrays do not have same size', async () => {
```

Comment:
DId you remove all the array discrepancy tests on purpose? It looked useful.

Reply:
In the past we had independent arrays in inputs, now, it is a list of "requests".